### PR TITLE
feat: improve rendering of summary emission form

### PR DESCRIPTION
**Pull request by matthieu-foucault**
Deleted redundant components

### CODE REVIEW

**Comment by matthieu-foucault** [comment]
@kriscooke I do need do update the end-to-end test making those screenshots, and there's something weird going on with the CSS generated by happo. Other than that the changes to the application code itself should be ready for review

File: app/containers/Forms/SummaryFormFieldTemplate.tsx [hunk 1]
```diff
@@ -67,29 +67,41 @@ const FormFieldTemplate: React.FunctionComponent<FieldTemplateProps> = ({
...
-          padding: 20px 20px 10px 0;
+          border: none;
+          display: inline-flex;
+          justify-content: space-around;
+          align-items: flex-start;
         }
+
         .summary-card .card-body {
           padding-top: 0;
           padding-bottom: 50px;
```
Comment (kriscooke) [inline app/containers/Forms/SummaryFormFieldTemplate.tsx:107]: Some change in this file has affected the size of the triangle exlamation mark icons that appear next to required property errors. Best starting point I can give is that sometimes setting flex properties can do funky unexpected things to the sizing of elements - check `flex-basis`, `flex-grow`, `flex-shrink` and similar:
Screen Shot 2020-07-23 at 12 38 20 PM
Screen Shot 2020-07-23 at 12 41 22 PM

Comment (kriscooke) [reply app/containers/Forms/SummaryFormFieldTemplate.tsx:107]: Also: 
Screen Shot 2020-07-23 at 12 48 52 PM

Comment (matthieu-foucault) [reply app/containers/Forms/SummaryFormFieldTemplate.tsx:107]: Looks good on Firefox. I'll add some happo screenshots with validation errors to make sure we're checking that CSS across browser

Comment (matthieu-foucault) [reply app/containers/Forms/SummaryFormFieldTemplate.tsx:107]: (actually on Firefox, the warning signs are not displayed at all)

**Pull request merged**